ttest.d.ts

```diff
@@ -0,0 +1,30 @@
+declare function ttest(
```

Comment:
I don't see the corresponding `ttest.js` file, is this types file still necessary?

Reply:
It is, this is for the `ttest` package we are using in `lib/generate-tstats`